sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py

```diff
@@ -381,6 +381,9 @@ def test_recover_and_purge(self, client, **kwargs):
     @all_api_versions()
     @client_setup
     def test_async_request_cancellation_and_deletion(self, client, **kwargs):
+        if self.is_live:
+            pytest.skip("Skipping by default because of operation flakiness in pipeline testing")
```

Comment:
Consider linking the issue here for the benefit of the parallel universe in which the issue gets fixed but the fixer forgets to unskip this test 😸